src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userservice = new UserService();
+
+        if (userservice.findByEmail(email) == null) {
+            return false;
+        } else if ((email == userservice.findByEmail(email).getEmail())
+                && password == userservice.findByEmail(email).getPassword()) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Let's also reorganize our logic just a bit, because it is hard to read at the moment
to avoid complications in if & else, we can do something like this:
```java
if (// user not null
    // user email is fine
    /*user password is fine*/) {
    return true;
}
return false;
```